tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -209,31 +209,26 @@ def test_imul_scalar(xp, scalar, device, shape, dtype):
     return lhs
 
 
-@chainerx.testing.numpy_chainerx_array_equal()
+@chainerx.testing.numpy_chainerx_array_equal(strides_check=False)
```

Comment:
Is it not easy to fix strides?

Reply:
It is a little difficult.
```py
import numpy  # numpy==1.16.1

def f(a):
    b = a / 1
    print(a.shape, a.strides, b.strides)

val = 0

f(numpy.full((0, 1, 1), val))
f(numpy.full((1, 0, 1), val))
f(numpy.full((1, 1, 0), val))
f(numpy.full((0, 1, 1), val).T)
f(numpy.full((1, 0, 1), val).T)
f(numpy.full((1, 1, 0), val).T)

f(numpy.full((0, 2, 2), val))
f(numpy.full((2, 0, 2), val))
f(numpy.full((2, 2, 0), val))
f(numpy.full((0, 2, 2), val).T)
f(numpy.full((2, 0, 2), val).T)
f(numpy.full((2, 2, 0), val).T)
```
The above code outputs as follows:
```
(0, 1, 1) (8, 8, 8) (8, 8, 8)
(1, 0, 1) (8, 8, 8) (0, 8, 8)
(1, 1, 0) (8, 8, 8) (0, 0, 8)
(1, 1, 0) (8, 8, 8) (0, 0, 8)
(1, 0, 1) (8, 8, 8) (0, 8, 8)
(0, 1, 1) (8, 8, 8) (8, 8, 8)
(0, 2, 2) (32, 16, 8) (32, 16, 8)
(2, 0, 2) (16, 16, 8) (0, 16, 8)
(2, 2, 0) (16, 8, 8) (0, 0, 8)
(2, 2, 0) (8, 16, 32) (8, 16, 32)
(2, 0, 2) (8, 16, 16) (8, 32, 16)
(0, 2, 2) (8, 8, 16) (16, 8, 0)
```

Reply:
OK.